content/influxdb/v2.0/get-started.md

```diff
@@ -347,6 +347,12 @@ user, and bucket.
 The setup process is available in both the InfluxDB user interface (UI) and in
 the `influx` command line interface (CLI).
 
+{{% note %}}
+#### Permissions for the initial user
+The user created in the InfluxDB setup process has **full read and write access to
+all organizations** in the database.
```

Comment:
Would it make sense to add another section under "Next Steps" below for creating new users? To remind people to think about whether or not they really want to be using a superuser-token to collect data.

Reply:
Good thought. I'll add that.